template-for-repository/proofs/Makefile.common

```diff
@@ -112,6 +112,9 @@ CBMCDIR ?= $(PROOF_ROOT)/cbmc
 CBMCFLAGS += --unwind 1 $(CBMC_UNWINDSET)
 
 # Additional CBMC flags used for property checking
+ifneq ($(EXTERNAL_SAT_SOLVER),)
```

Comment:
Please do
```
ifneq ($(strip $(EXTERNAL_SAT_SOLVER)),)
```
instead, as some of the Makefiles are written such that variables can get assigned whitespace.